semantic_conventions/trace/database.yaml

```
@@ -385,6 +385,47 @@ groups:
           The collection being accessed within the database stated in `db.name`.
         examples: [ 'customers', 'products' ]
 
+  - id: db.elasticsearch
+    prefix: db.elasticsearch
+    type: span
+    extends: db
+    brief: >
+      Call-level attributes for Elasticsearch
+    attributes:
+      - id: doc_id
+        type: string
+        requirement_level:
+          conditionally_required: when the request targets a specific document by id
+        tag: call-level-tech-specific
```

otherwise I think the markdown generation will try to add this to the existing `call-level-tech-specific` table, which I think may be why one of the CI checks is failing
```suggestion
        tag: call-level-tech-specific-elasticsearch
```